src/main/java/sirius/biz/tycho/dashboards/KeyMetric.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.tycho.dashboards;
+
+import sirius.biz.analytics.metrics.MetricQuery;
+import sirius.kernel.commons.Amount;
+import sirius.kernel.commons.NumberFormat;
+import sirius.web.services.JSONStructuredOutput;
+
+import javax.annotation.Nullable;
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Represents a key metric to be shown on a metric dashboard.
```

Comment:
Maybe expand this description a bit and link to it from the key metric providers, etc via a `@see`